### DISCUSSION TITLE
Lost fonts

### DISCUSSION THREAD

**Discussion by A-Winters** [SubtitleEdit/subtitleedit#5468] [Q&A] [answered]
I've copied my .ass files to my new laptop and installed Subtitle Edit on there too.
In the ASS styles window a lot of the fonts in the drop down menu are missing on this laptop.
I don't know if they're fonts that I'd unknowingly imported on my old laptop or if some SE fonts just don't work on Windows 11.

Is there a way to transfer them to SE on my new laptop from my old one, please?
I've tried looking for options to download more fonts, but I don't know what I'm doing or what I'm looking for.

Please help!

**Comment by LeonCheung** [comment, marked as answer]
Hmm, I don't think SE supports that yet for the time being. A tool to collect and pack used fonts is a nice idea.

**Reply by A-Winters** [reply]
Download and import font options would be great for customisation.

I really don't want to have to resize everything for the substituted fonts, so I tried exporting and importing styles, but it only worked on the font names and not the appearance as they haven't been added to the dropdown menu.

I'm about to mux the subs to videos on my old laptop so I can see if importing the subs from videos makes any difference.
I'll report back with results.

Is there anything in the AppData I could just copy and paste across? I've had a look, but I don't know what I'm looking for.

Update: My idea of transporting the subs in a video didn't add the fonts either.
Copying and pasting files might be my last hope.

**Reply by LeonCheung** [reply]
@A-Winters Maybe you can try copying this entire folder:  C:\WINDOWS\Fonts. For mine, it's less than 1Gb. And overwrite it to the laptop folder (don't worry, it won't remove existing fonts). Hope it won't cost you too much time.

**Reply by A-Winters** [reply]
This worked, thank you!
I'll have to put fonts in the download folder with instructions when I upload them online.
